### PR TITLE
Add a server info panel

### PR DESCRIPTION
**Pull request by uniboi**
image

image

I made this since #335 got stale and outdated.

This doesn't update server info when connecting to servers with commands but I'm confident the average user won't do that.

@GeckoEidechse this pr adds two vanilla files

- resource/ui/menus/ingame_mp.menu
- scripts/vscripts/ui/menu_ingame.nut

### CODE REVIEW

**Review by ASpoonPlaysGames — CHANGES_REQUESTED** [review]
files added in this PR that were in vanilla before should be committed without changes first with respawn as the author @GeckoEidechse (menu_ingame.nut, ingame_mp.menu)

**Comment by GeckoEidechse** [comment]
The menu that gets replaced is the one that shows challenges needed to complete to earn extra merits, right?

Also should we maybe add server region as well? ^^

**Comment by uniboi** [comment]
updated
image

**Review by F1F7Y — CHANGES_REQUESTED** [review]
Hi, hello my second favorite german. New review just dropped.

Also the margin should be the same on the text cause this looks funny
image

File: Northstar.Client/mod/resource/ui/menus/panels/server_info.res [hunk 1]
```diff
@@ -0,0 +1,68 @@
+resource/ui/menus/panels/server_info.res
```
[the remark this answers is not in the document]
Comment (uniboi) [reply Northstar.Client/mod/resource/ui/menus/panels/server_info.res:1]: wdym it's formatted cool

File: Northstar.Client/mod/resource/ui/menus/ingame_mp.menu [hunk 2]
```diff
@@ -0,0 +1,97 @@
+#base "combo_buttons.res"
```
[the remark this answers is not in the document]
Comment (ASpoonPlaysGames) [reply Northstar.Client/mod/resource/ui/menus/ingame_mp.menu:1]: Tbh this is a (modified) respawn file, so idk if it's worth formatting

File: Northstar.Client/mod.json [hunk 3]
```diff
@@ -41,6 +41,10 @@
 			"Name": "modlist_reverse",
 			"DefaultValue": "0",
 			"Flags": 16777216
+		},
+		{
+			"Name": "ns_connected_server",
+			"DefaultValue": ""
 		}
```
Comment (ASpoonPlaysGames) [inline Northstar.Client/mod.json:48]: Weren't we going to make server name and stuff FCVAR_REPLICATED which would make this unnecessary?

Comment (uniboi) [reply Northstar.Client/mod.json:48]: is there a pr for that?

Comment (ASpoonPlaysGames) [reply Northstar.Client/mod.json:48]: Don't think so, but it was talked about a lot in discord and stuff iirc?

Comment (GeckoEidechse) [reply Northstar.Client/mod.json:48]: Wait they are? I thought that PR never got merged?

**Comment by GeckoEidechse** [comment]
@Jan200101 said you should update this PR :>

**Comment by ASpoonPlaysGames** [comment]
@uniboi is this PR stale?

**Comment by uniboi** [comment]
Kinda forgot about it.
Basically I'm waiting for the server name and description convars to get networked but that needs a version gate

**Comment by GeckoEidechse** [comment]
Networked convar stuff is still basically waiting on us doing a version gate. The thing with server info panel is that now with progression we kinda make use of that HUD element now that would get replaced...

**Comment by Zanieon** [comment]
Heads up that this will conflict entirely with #756 since it's the same menu and panels being used.

**Pull request closed without merging**